core/src/main/scala/neotypes/Driver.scala

```diff
@@ -8,14 +8,42 @@ import org.neo4j.driver.v1.{AccessMode, Driver => NDriver}
 import scala.jdk.CollectionConverters._
 import scala.language.higherKinds
 
+/** neotypes driver for accessing a neo4j graph database
+  *
+  * @param driver neo4j driver
+  * @tparam F effect type for driver
+  */
 final class Driver[F[_]](private val driver: NDriver) extends AnyVal {
+
+  /** acquire session to a database in read mode
```

Comment:
A minor comment about capitalization of first letter. It would be great to make it consistent with previous PRs